Bindings/Packages/com.virtualmaker.bindings/Editor/NetworkPropertyEditor.cs

```diff
@@ -0,0 +1,40 @@
+#if UNITY_NETCODE_GAMEOBJECTS
+using UnityEditor;
+using UnityEngine;
+
+namespace VirtualMaker.Bindings.Editor
+{
+    [CustomPropertyDrawer(typeof(NetworkProperty<>), true)]
+    public class NetworkPropertyEditor : PropertyEditor
+    {
+        private const string k_InternalValue = "m_InternalValue";
```

Comment:
```suggestion
        private const string InternalValuePropertyName = "m_InternalValue";
```

- Don't use k_
- I'd just inline the string